CMakeLists.txt

```diff
@@ -152,6 +152,12 @@ add_subdirectory("sail_runtime")
 # Sail configuration
 add_subdirectory("config")
 
+#JSON validation
+set(valijson_BUILD_TESTS OFF CACHE BOOL "don't build valijson tests")
```

Comment:
Since both of these have CMake support, how about using [FetchContent](https://cmake.org/cmake/help/latest/module/FetchContent.html)?